osquery/tables/system/windows/logical_drives.cpp

```diff
@@ -0,0 +1,84 @@
+#include "osquery/core/windows/wmi.h"
+#include <osquery/tables.h>
+
+namespace osquery {
+namespace tables {
+
+QueryData genLogicalDrives(QueryContext& context) {
+  Row r;
```

Comment:
Any reason for the `Row r` to be this high in scope? Perhaps pull it inside of your `for` below closer to where it's used to ensure you're making a new `Row` for each of the entires you're processing.